controllers/storagecluster/cephcluster.go

```diff
@@ -114,6 +114,13 @@ func (obj *ocsCephCluster) ensureCreated(r *StorageClusterReconciler, sc *ocsv1.
 		sc.Spec.StorageDeviceSets[i].Config.TuneSlowDeviceClass = false
 		sc.Spec.StorageDeviceSets[i].Config.TuneFastDeviceClass = false
 
+		// Use SSD deviceClass and TuneFastDeviceClass for all internal clusters where deviceClass is not provided by the customer
+		if !sc.Spec.ExternalStorage.Enable && sc.Spec.StorageDeviceSets[i].DeviceClass == "" {
+			sc.Spec.StorageDeviceSets[i].DeviceClass = DeviceTypeSSD
+			sc.Spec.StorageDeviceSets[i].Config.TuneFastDeviceClass = true
+			continue
+		}
+
 		diskSpeed, err := r.checkTuneStorageDevices(ds)
```

Comment:
I believe yes. Its still needed. For internal setups using custom `deviceClass` (like Managed Services), this new if block will be skipped.

Reply:
I see now, thanks. So this means for example if an AWS admin wants to set the TuneSlowDeviceClass, they could set the `deviceClass: ssd`, and then the TuneSlowDeviceClass would be set since AWS is not in the slice `TuneFastPlatforms`. Not an intuitive behavior, but I can live with that.